### PR TITLE
Feature/se 1119.add tests for term deposit

### PR DESCRIPTION
**Pull request by lorenzdec**
# Pull Request Checklist

- [ ] Read the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] Tests pass
- [ ] Raised the PR against the `develop` branch

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: sdk/tests/tutorials/instruments/test_complex_instruments.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import lusid
 import lusid.models as models
 from datetime import datetime
-from utilities import TestDataUtilities
+#from utilities import TestDataUtilities
```
Comment (jyaffe) [inline sdk/tests/tutorials/instruments/test_complex_instruments.py:6]: If this import is no longer needed then the line should be deleted, although it looks like it is required (line 12) so best to remove the `#`.

File: sdk/tests/tutorials/instruments/test_complex_instruments.py [hunk 2]
```diff
@@ -106,3 +106,31 @@ def test_create_fx_option(self):
         self.assertEqual(
             saved_fx_option.is_delivery_not_cash, fx_option.is_delivery_not_cash
         )
+
+        # to be inserted here
```
Comment (jyaffe) [inline sdk/tests/tutorials/instruments/test_complex_instruments.py:110]: This comment should be removed

File: sdk/tests/tutorials/instruments/test_complex_instruments.py [hunk 3]
```diff
@@ -106,3 +106,31 @@ def test_create_fx_option(self):
...
+                settleDays=1,
+                resetDays=0
+                ),
+            rate=0.03,
+            instrument_type="TermDeposit"
+        )
+
+        # Assert instrument was created
+        self.assertIsNotNone(term_deposit)
+
```
Comment (jyaffe) [inline sdk/tests/tutorials/instruments/test_complex_instruments.py:135]: Ideally there would be a further set of assertions under this test, as is the case in other tests above and in the C# example.

**Review by jyaffe — CHANGES_REQUESTED** [review]
Good initial commit but a bit of cleanup is needed as well as further assertions within the test. 

Please can you also update the PR description and checklist? It will help us quickly understand this PR if we need to come back and review it in the future.

File: sdk/tests/tutorials/instruments/test_complex_instruments.py (hunk 1 above)
Comment (lorenzdec) [reply sdk/tests/tutorials/instruments/test_complex_instruments.py]: Initially did this to debug, have now removed the # so it is back in the newer version.

File: sdk/tests/tutorials/instruments/test_complex_instruments.py (hunk 2 above)
Comment (lorenzdec) [reply sdk/tests/tutorials/instruments/test_complex_instruments.py]: Has been removed now

File: sdk/tests/tutorials/instruments/test_complex_instruments.py (hunk 3 above)
Comment (slemasne) [reply sdk/tests/tutorials/instruments/test_complex_instruments.py:141]: Usually I would agree more assertions the better.
Looks like in this example, we'll keep the assertions light, as these operations have already been tested in core LUSID code:
https://github.com/finbourne/lusid-sdk-python-preview/pull/88#issuecomment-802686799
(FYI - @rizwansaeed )

**Pull request merged**